commercial/app/CommercialLifecycle.scala

```diff
@@ -21,7 +23,12 @@ class CommercialLifecycle(
   akkaAsync: AkkaAsync,
   feedsFetcher: FeedsFetcher,
   feedsParser: FeedsParser,
-  industries: Industries)(implicit ec: ExecutionContext) extends LifecycleComponent with Logging {
+  industries: Industries) extends LifecycleComponent with Logging {
+
+  // This class does work that should be kept separate from the EC used to serve requests
+  implicit private val ec = ExecutionContext.fromExecutorService(
+    Executors.newFixedThreadPool(10)
```

Comment:
Is there a particular rationale for 10, or is this likely to be tweaked later? What instance types does commercial run on?

Reply:
We expect to tune that number based on the performance under actual load. We are currently on a `LargeBurstable` stack using `m5.large` as per https://github.com/guardian/platform/pull/1389/files

Reply:
It's most likely Commercial will run on `t3.large`, `m5.large` being an instance type that'll be picked up if it runs on spots. We've reserved t3.large so usually they'll be running on those.